test/dockerized-integration-tests.sh

```diff
@@ -9,4 +9,4 @@ else
 fi
 
 # Core integration tests
-docker-compose run -T -e HEADLESS=true --rm ${CONTAINER} vendor/bin/phpunit --configuration test/phpunit.xml
+docker-compose run -T -e HEADLESS=true --rm ${CONTAINER} vendor/bin/phpunit --configuration test/phpunit.xml --testsuite LorisModuleIntegrationTests $*
```

Comment:
This way will miss LorisCoreIntegrationTests as well.
Please add LorisCoreIntegrationTests into this file.

Reply:
@kongtiaowang It's done!